much-rails.gemspec

```diff
@@ -24,6 +24,8 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency("activerecord", ["> 5.0", "< 7.0"])
   gem.add_dependency("activesupport", ["> 5.0", "< 7.0"])
+  gem.add_dependency("much-boolean", ["~> 0.1.3"])
+  gem.add_dependency("much-not-given", ["~> 0.1.0"])
```

Comment:
Wasn't sure if I needed to explicitly add `much-not-given` as `assert` brings it in....but thought explicit > implicit.

Reply:
Yeah good call on this - explicit > implicit, also MuchRails doesn't _know_ that Assert brings this in, technically.